packages/embed/src/apple-pay/apple-pay.ts

```diff
@@ -1,11 +1,70 @@
 import { SubjectManager } from 'subjects'
+import { error, log, warn } from 'utils'
+
+export const APPLE_PAY_TIMEOUT = 1000
 
 declare global {
   interface Window {
     ApplePaySession?: ApplePaySession
   }
 }
 
+export const loadApplePaySdk = () => {
+  return Promise.race([
+    new Promise<boolean>((resolve) => {
+      if (typeof window === 'undefined') {
```

Comment:
Bail on initial SSR renders where the `window` object is not available. This gets picked up again in the client (see https://spider.plantly.dev/ as an example)